src/Util/Log/JUnit.php

```diff
@@ -229,6 +229,7 @@ public function startTestSuite(TestSuite $suite)
     {
         $testSuite = $this->document->createElement('testsuite');
         $testSuite->setAttribute('name', $suite->getName());
+        $testSuite->setAttribute('package', $suite->getName());
```

Comment:
http://llg.cubic.org/docs/junit/ says "Derived from testsuite/@name in the non-aggregated documents. optional".